<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdatePatientInfo</class>
 <widget class="QDialog" name="UpdatePatientInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1361</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:&quot;#111111&quot;;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #00d2ff;</string>
   </property>
   <property name="text">
    <string>Clinic Managemnet System - Update Patient</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #1c1c1c;  
border: 1px solid #333333;  
border-radius: 6px;
color: #ffffff;            
padding: 10px 15px;
font-size: 15px;
font-weight: bold;
transition: 0.45s ease;
}

QPushButton:hover {
background-color: #111111; 
border: 1px solid #00d2ff; 
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_patient_btn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>640</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #1c1c1c;  
border: 1px solid #333333;  
border-radius: 6px;
color: #ffffff;            
padding: 10px 15px;
font-size: 15px;
font-weight: bold;
transition: 0.45s ease;
}

QPushButton:hover {
background-color: #111111; 
border: 1px solid #00d2ff; 
}</string>
   </property>
   <property name="text">
    <string>Update Patient Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Token Nmber</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_patient_input">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #1c1c1c;  
border: 1px solid #333333;  
border-radius: 6px;
color: #ffffff;            
padding: 10px 15px;
font-size: 15px;
font-weight: bold;
transition: 0.45s ease;
}

QPushButton:hover {
background-color: #111111; 
border: 1px solid #00d2ff; 
}</string>
   </property>
   <property name="text">
    <string>ðŸ”Ž</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_name">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>330</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_age">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
padding: 5px 0px;</string>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
padding: 5px 0px;</string>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctors_name">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>380</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_gender">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>480</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>530</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
padding: 5px 0px;</string>
   </property>
   <property name="text">
    <string>Disease</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
padding: 5px 0px;</string>
   </property>
   <property name="text">
    <string>Patient Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_disease">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>530</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
padding: 5px 0px;</string>
   </property>
   <property name="text">
    <string>Doctor's Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>230</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font-size: 28px</string>
   </property>
   <property name="text">
    <string>Patient Info</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>611</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1c1c1c;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_2</zorder>
  <zorder>back_btn</zorder>
  <zorder>add_patient_btn</zorder>
  <zorder>label_9</zorder>
  <zorder>search_patient_input</zorder>
  <zorder>search_btn</zorder>
  <zorder>patient_name</zorder>
  <zorder>patient_age</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>doctors_name</zorder>
  <zorder>patient_gender</zorder>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>patient_disease</zorder>
  <zorder>label_3</zorder>
  <zorder>label_10</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
